rpc/srv/dsHdmiIn.c

```diff
@@ -688,6 +721,63 @@ IARM_Result_t _dsHdmiInInit(void *arg)
             INT_INFO("Port %s: Exception in Getting the HDMI2 EDID allm support from persistence storage..... \r\n", "HDMI2");
             m_edidallmsupport[dsHDMI_IN_PORT_2] = true;
         }
+
+       // Getting the edidvrrEnable value from persistence upon bootup
+        std::string _VRRSupport("TRUE");
+        try {
+            _VRRSupport = device::HostPersistence::getInstance().getProperty("HDMI0.vrrEnable");
+            if(_VRRSupport == "TRUE")
```

Comment:
_VRRSupport is initialized to true so why we checking here for TRUE which will always be the success case